### PR TITLE
Accessible self sovereign identity

### PR DESCRIPTION
**Pull request by victormartinez-work**
Andrew Slack and Victor Martinez (SICPA)

Digital identity is a foundational building block in the infrastructure of modern society, for it to function as a public good it must be available and accessible to all. As such inclusive design of self-sovereign identity systems is needed to ensure safe and consistent use.

The experience and security of any system operated by people depends on the information conveyed through user interfaces, the response of the users, and the interpretation of their actions. Existing interaction patterns in verifiable credential wallets tend towards visual-centric models and rely on inconsistent representations of data. This paper explores design patterns to manage and exchange verifiable data in more accessible ways.

We intend to look at:

- how systems can be designed to support accessible issuance and verification flows
- how and where existing accessibility standards can be incorporated
- how verifiable data representations can be designed for appropriation by wallets

### CODE REVIEW

**Comment by elenachachkarova** [comment]
@victormartinez-work, thank you for your submission. Ideally, topic PRs should also include an abstract in the #topical-listing section of the README.md (see: https://github.com/WebOfTrustInfo/rwot11-the-hague/blob/master/advance-readings/README.md#topical-listing ) and a link in the Alphabetical Listing (see: https://github.com/WebOfTrustInfo/rwot11-the-hague/tree/master/advance-readings#alphabetical-listing). If possible, could you commit those as well and I will merge the PR. Thank you.

File: advance-readings/accessible-self-sovereign-identity.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Overlays Capture Architecture (OCA) offers a solution to harmonization between data models and data representation formats. In other words, a way to define the semantics of a data model. 
+
+OCA represents a schema as a multi-dimensional object consisting of a stable schema base and linked overlays, data objects that provide additional extensions, coloration, and functionality to the base object. Overlays are cryptographically-linked objects that provide layers of task-oriented contextual information to a Capture Base. 
+
+OCA classifies four types of overlays : Semantic overlays, Inputs overlays, transformation, and presentations overlays. 
+
+This paper would like to explore how OCA overlays can be used  (in particular presentation overlays)  or extend the specification to include data representations accessible to all.
+
+
+## Work behond RWoT11 - Improving Annoncreds accesability
```
Comment (jandrieu) [inline advance-readings/accessible-self-sovereign-identity.md:53]:
```suggestion
## Work behond RWTO11 - Improving Annoncreds accesability
```

**Comment by victormartinez-work** [comment]
> @victormartinez-work, thank you for your submission. Ideally, topic PRs should also include an abstract in the #topical-listing section of the README.md (see: https://github.com/WebOfTrustInfo/rwot11-the-hague/blob/master/advance-readings/README.md#topical-listing ) and a link in the Alphabetical Listing (see: https://github.com/WebOfTrustInfo/rwot11-the-hague/tree/master/advance-readings#alphabetical-listing). If possible, could you commit those as well and I will merge the PR. Thank you.

Dear @elenachachkarova  I have updated the readme file accordingly.

**Comment by elenachachkarova** [comment]
@victormartinez-work, thank you for making the change, I merged the PR.

**Pull request merged**